Demo/MozillaSocial-iOS/MoSoContent/LoginView.swift

```diff
@@ -0,0 +1,24 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import SwiftUI
+
+struct LoginView: View {
+    let auth = Auth()
+
+    var body: some View {
+        Button("Login!") {
+            Task {
+                guard let clientEntity = await auth.registerApp() else {
+                    print("Client Entitiy Nil")
+                    return
+                }
+                auth.signIn(client: clientEntity)
+            }
+        }
+    }
+
+
```

Comment:
nit: perhaps we can remove the extra spaces? @cyndichin wondering why these are not captured by `SwiftLint`

Reply:
ah this is due to the demo folder not being observed by swiftlint. Will need to add `  - ../../Demo` under included section. Since we're modifying the code in Demo, I wonder if we need to have `MoSoContent` folder since Demo seems to act as the main app.